websites/T/Tver/metadata.json

```diff
@@ -0,0 +1,98 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "223277933521928203",
+		"name": "ryunt"
+	},
+	"service": "Tver",
+	"description": {
+		"en": "TVer is a Japanese ad-supported video on demand (AVOD) service. It allows viewers to watch TV programs they missed during their initial broadcast for free for one week.",
```

Comment:
is it TVer or Tver, cause the service and folder name has another capitalization then in the description.

Reply:
looks like it is `TVer` based on the page's title

Reply:
Fixed it to TVer, thanks for the heads-up!